Vagrantfile

```diff
@@ -347,6 +347,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         end
         
         config.ssh.insert_key = false
+
+        $enablePswSSH = <<-SCRIPT
+        sudo sed -i "/^[^#]*PasswordAuthentication[[:space:]]no/c\PasswordAuthentication yes" /etc/ssh/sshd_config
+        sudo service sshd restart
+        SCRIPT
+
+        host.vm.provision "shell", inline: $enablePswSSH
```

Comment:
This should run only for the box you use for libvirt. For the other one is not currently needed.